doc-content/enterprise-only/openshift/operator-deploy-pim-proc.adoc

```diff
@@ -0,0 +1,60 @@
+[id='operator-deploy-pim-proc']
+= Setting Process Instance Migration configuration for the environment
+
+You can use the operator to deploy the Process Instance Migration (PIM) service. You can use the PIM service to define the migration between two different process definitions, known as a migration plan. You can then apply the migration plan to the running process instance in a specific KIE Server. 
+
+The PIM service uses a database server for its operation.
+
+.Prerequisites
+
+* You completed basic configuration of a {PRODUCT} environment using {operator_name} in the installer wizard according to the instructions in <<operator-deploy-basic-proc>>.
+
+.Procedure
+. If the *Installation*, *Security*, *Console*, *KIE Servers*, or *Smart Router* tab is open, click *Next* until you view the *Process Instance Migration* tab.
+. In the *Database type* field, select the database that the PIM service must use. The following values are available:
+** `mysql`: A MySQL server, created in a separate pod.
+** `postgresql`: A PostgreSQL server, created in a separate pod. Use this setting unless you have a specific reason to use any other setting.
+** `h2`: A built-in `h2` database engine that does not require a separate pod.
+** `external`: An external database server. 
+. If you selected any database except `external`, a Persistent Volume Claim will be created to store the database. Optionally, set configuration parameters of the persistent volume to create:
+** In the *Size* field, enter the size of the persistence volume
+** In the *StorageClass name* field, enter the storage class name for the persistent volume
+. If you selected the `external` database and you want to use any database server except PostgreSQL or MySQL, you must provide a {KIE_SERVER} extension image with the database server driver according to instructions in <<externaldb-build-proc>>. The PIM service can use the {KIE_SERVER} extension image. To configure the PIM service to use this extension image, make the following changes:
```

Comment:
The kie-server extension images are not related to PIM images. PIM images are built differently. It's enough to include in the image the driver somewhere accessible and add it to the classpath using  the JBOSS_KIE_EXTRA_CLASSPATH env var.

Reply:
Unfortunately, we can't make it accessible *without* a custom image (or some configmap approach). I will therefore change the doc to list only MySQL/PostgreSQL/MariaDB as possible DB servers for PIM, and create an RHPAM issue about absence of a way to provide an external driver. I will create a review task for you once that is done.